package.json

```diff
@@ -122,6 +122,9 @@
       "dprint fmt"
     ]
   },
+  "overrides": {
+    "liquidjs": "^10.4.0"
+  },
```

Comment:
Not ideal, especially as we are not using liquidjs.

Reply:
:disappointed: